### DISCUSSION TITLE
Cara mengecek R-Adjusted Squared

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#395] [Q&A] [answered]
Permisi saya waktu itu membuat model random forest nah untuk melihat goodness of fit/R adjusted Squared itu dari model random forest gimana ya? dan apakah ada fungsi dari R-Adjusted Squared untuk mengecek di berbagai model?
![image](https://user-images.githubusercontent.com/78478403/194790958-761da9a1-86ba-4bc6-a4ad-c9ccace175f0.png)
Dan var explained itu apa ya 

Terima kasih

**Comment by takdirzd** [comment, marked as answer]
Setau saya kak kalau modelnya random forest, itu bisa dilihat R-Squarednya untuk mengukur goodness of fitnya.. Kalau maksud kakak R-Squarednya, berdasarkan yang pernah saya coba, saya menggunakan sebuah fungsi dibawah ini. Untuk melihat R-Squarednya, kakak bisa menggunakan fungsi dibawah ini :

![image](https://user-images.githubusercontent.com/88231577/194813750-a30da20d-d42c-445b-83d6-1409463c1ffc.png)

Referensi  = https://algotech.netlify.app/blog/interpreting-black-box-regression-model-with-lime/

Sehingga nanti untuk outputnya seperti ini : 
![image](https://user-images.githubusercontent.com/88231577/194813920-9eedd661-a9a0-4599-baf9-2116e51d0470.png)

Semoga membantu..

**Reply by daryraa** [reply]
ohh baik kak terima kasih
